app/models/user.rb

```diff
@@ -65,6 +65,7 @@ def student
   has_many :applications, through: :teams
   has_many :todos, dependent: :destroy
   has_many :comments, dependent: :destroy
+  has_many :postal_addresses
```

Comment:
Why `has_many`? I'd personally have gone for `has_one` 🤔 

I don't think we need more than one `address` per `student`.